coredb-operator/Dockerfile

```diff
@@ -1,4 +1,17 @@
-FROM gcr.io/distroless/static:nonroot
-COPY --chown=nonroot:nonroot ./controller /app/
+FROM clux/muslrust:stable AS builder
+
+COPY . .
+
+RUN cargo build --release
+
+FROM alpine:latest
+
+RUN adduser -D nonroot
+
+COPY --chown=nonroot:nonroot --from=builder target/x86_64-unknown-linux-musl/release/controller /app/controller
+COPY --chown=nonroot:nonroot --from=builder target/x86_64-unknown-linux-musl/release/crdgen /app/crdgen
+
+USER nonroot
+
```

Comment:
Although rust compiles to a binary, by default that binary has dependencies in the system like openssl and libc. If we build with MUSL we can avoid that - like clux did in the controller example.